index.html

```diff
@@ -152,6 +163,14 @@
             return mortality
         }
 
+        function getDeadPerMillion(data, country) {
+            var dead = []
+            for (var day of data[country]) {
+                if (day["deaths"] < 1) continue
```

Comment:
I would prefer "Days since 25 confirmed" to keep it comparable with other graphs